packages/web3-utils/src/errors.ts

```diff
@@ -83,3 +83,9 @@ export class InvalidDenominatorError extends Web3Error {
 		super(value, 'denominator must be number power of 10');
 	}
 }
+
+export class NibbleWidthError extends Web3Error {
+	public constructor(value: unknown) {
```

Comment:
```suggestion
	public constructor(value: string) {
```

Based on how you use the error:

```typescript
throw new NibbleWidthError(`value: "${value}", nibbleWidth: "${nibbleWidth}"`);
```